src/solidity-frontend/solidity_grammar.h

```diff
@@ -0,0 +1,190 @@
+#ifndef SOLIDITY_GRAMMAR_H_
+#define SOLIDITY_GRAMMAR_H_
+
+#include <map>
+#include <string>
+#include <nlohmann/json.hpp>
+
+#define ENUM_TO_STR(s) case s: { return #s; }
```

Comment:
Does this macro have to be defined on this header? Can you move it to the cpp file where it's used?

Reply:
Done.
Moved to the corresponding cpp file.